src/validation/validateStatsDate.js

```diff
@@ -0,0 +1,31 @@
+import BaseJoi from 'joi';
+import Extension from 'joi-date-extensions';
+import JoiCountryExtension from 'joi-country-extension';
+import setLanguage from 'utils/international';
+
+const Joi = BaseJoi.extend(Extension, JoiCountryExtension);
+
+const validateStats = (req, res, next) => {
+  const { preferedLang } = req.user;
+  const schema = Joi.object().keys({
+    from: Joi.date().format('YYYY-MM-DD').required()
+      .error(() => ({
+        message: 'fromDate'
+      })),
+    until: Joi.date().format('YYYY-MM-DD').required()
+      .error(() => ({
+        message: 'untilDate'
+      })),
```

Comment:
just as i suggested before 
```
 from: Joi.date().format('YYYY-MM-DD').required()
      .error(() => ({
        message: 'fromDate'
      })),

    until: Joi.date().format('YYYY-MM-DD').min(Joi.ref('from').required()
        .error(() => ({
          .... error and close
```
   something like this can do the magic